.github/CODEOWNERS

```diff
@@ -6,6 +6,8 @@
 
 # Tendermint2 (Gno version).
 tm2/*                        @gnolang/tech-staff
+tm2/pkg                      @jaekwon @moul
```

Comment:
Reviews by @jaekwon and @moul are mandatory for merging, while others are encouraged.